django_mailbox/models.py

```diff
@@ -362,7 +363,7 @@ def _get_dehydrated_message(self, msg, record):
     def _process_message(self, message):
         msg = Message()
         if STORE_ORIGINAL_MESSAGE:
-            msg.eml.save('%s.eml' % uuid.uuid4(), ContentFile(message), save=False)
+            msg.eml.save('%s.eml' % uuid.uuid4(), ContentFile(force_text(message)), save=False)
```

Comment:
Ahh --  I wonder why I hadn't encountered this before; thanks for posting a patch!  

Just one suggestion, though, I *think* that the proper solution for this is probably to ask the message object to convert itself to a string using [`.as_string(unixfrom=True)`](https://docs.python.org/2/library/email.message.html#email.message.Message.as_string), rather than asking Django to stringify the object.  Although using `force_text` will work, it's a little bit like trying to fix a watch with a hammer.